core/services/keystore/csa.go

```diff
@@ -91,6 +110,7 @@ func (ks *CSA) Unlock(password string) error {
 	}
 
 	for i := range keys {
+		logger.Infow("Unlocked CSA Key")
```

Comment:
Do we want to add some info about the key that was unlocked e.g. address?

Reply:
Also changed to debug like the OCR keys